settings/_settings.global.scss

```diff
@@ -2,17 +2,23 @@
 //   #GLOBAL
 //   ======================================================================== */
 
-// Project-level settings and setup.
+// The global settings file contains any project-wide variables.
+
 
 // Base typographical styles and baseline grid. You need to define these values
 // in pixels: inuitcss will convert them to more appropriate units.
+
 $inuit-global-font-size:    16px !default;
 $inuit-global-line-height:  24px !default;
 
+
 // Do we need to support IE8?
+
 $inuit-oldIE: false !default;
 
+
 // Standardise some UI treatments.
+
 $inuit-global-radius:       3px !default;
```

Comment:
I want to discuss this file in general anyway—I think it should be made into `_example.settings.global.scss`, so that people can store all of their global settings (e.g. border radius stuff) in it, or we rename it to `_settings.core.scss`.

For now, let’s leave this stuff in there—once we’ve decided on the above we can rip it out :)